### ISSUE TITLE
[EngSys][Feature] Offer an easy way to let us know if a commit ID is part of the main branch

### ISSUE THREAD

**Issue by ArthurMa1978** [Azure/azure-sdk-tools#9372]
Currently, it's difficult to determine if an SDK PR that generates code from TypeSpec is based on the main branch of our spec repo.

Since we can already support invoke the [GitHub API](https://github.com/Azure/azure-sdk-tools/blob/59d869dd3abdbc1aa6d8d3935f9c52fe3ba6d37a/eng/common/scripts/Invoke-GitHubAPI.ps1#L456-L474) to retrieve commit information, could EngSys provide an easy way to display this information in any SDK PRs?

**Comment by weshaggard** [comment]
What is it exactly you think would be needed here? Do you simply want to see if the sha in the tsplocation.yml file is merged into main or not?  And how would you expect a PR to be marked with this information?

@raych1 don't we already have a check in the pipeline that validates this for releases?

**Comment by ArthurMa1978** [comment]
Yes, I want to easily know it which it's not straightforward to get it now. 
The pipeline validates the commit ID during release, but this validation does not occur during a PR review.

**Comment by weshaggard** [comment]
If we did the validation in the PR logs and outputted a message there would that be enough?

**Comment by raych1** [comment]
@ArthurMa1978 I want to confirm that if you want to see if this info in the `spec PR` review or `SDK PR` review?
